### DISCUSSION TITLE
エッヂ内のワードを学習したスクリプトが無差別に荒らしレスを行っているため、スレが機能不全に陥っています。

### DISCUSSION THREAD

**Discussion by gagagade** [edginer/eddiner#41] [General]
他のスレにもいるようですが、例としては下記スレがあります。

今年のサンマについて語るスレ
http://bbs.eddibb.cc/test/read.cgi/liveedge/1725818498/
のID:AmvfzGagpが、同一IDにもかかわらず、違うワッチョイになっていることからスクリプトと思われます。

ご対処をよろしくおねがいします。

以下、現時点で分かっていることのまとめのようです。

228 エッヂの名無し (L20 L78R-67l3)  2024/09/09(月) 05:56:08.184 ID:kp5rTGHt4
AIの書き込みでわかっていること
・IDが複数あり、必死チェッカーだと判別は難しい
・現状わかっているIDはL1の-41、L20の-61
・判別方法は「そのレスが何秒以内に書き込まれたか」程度
・レベルも1~6~20と複数確認されている
・名前欄が必ず「エッヂの名無し」
・ほぼすべてのスレに出没する
・学習内容がエッヂの中身っぽいので普通にレスバ仕掛けてくる
引用・http://bbs.eddibb.cc/test/read.cgi/liveedge/1725823574/

**Comment by 7scnBChdW** [comment]
今日も活動中のようです
https://www.kyodemo.net/sdemo/b/e_e_liveedge/?bs=hi&k=BNHhowJh4
https://www.kyodemo.net/sdemo/b/e_e_liveedge/?bs=hi&k=Nyu6ibQIu
https://www.kyodemo.net/sdemo/b/e_e_liveedge/?bs=hi&k=LYEVbF3lr
https://www.kyodemo.net/sdemo/b/e_e_liveedge/?bs=hi&k=OkCyTjG%2F6
